lib/roo_on_rails/sidekiq/process_scaling.rb

```diff
@@ -15,14 +15,15 @@ def initialize(queue_latencies)
 
       def current_processes
         ::Sidekiq::ProcessSet.new.count do |process|
+          process['quiet'] == 'false' &&
```

Comment:
Eww this feels nasty... had a look at the Sidekiq code and it's a string because it's just the result of an `hmget` but I'd be inclined to either do `.to_s == 'false'` or convert it to a Boolean before comparing it just to ensure that if it does ever get 'fixed' to be a real Boolean this code continues to work.